### DISCUSSION TITLE
Is this kit for React Devs only?

### DISCUSSION THREAD

**Discussion by itsmnthn** [rainbow-me/rainbowkit#323] [Q&A] [answered]
Is this kit for React Devs only?

**Comment by nickbytes** [comment, marked as answer]
RainbowKit currently only works with React. [wagmi](https://wagmi.sh/) recently shipped their 0.3.x update where they've separated the `core` and `react` packages. This opens up the possibility for wagmi to build libraries for other clients, like Vue or Svelte. If wagmi added support for these libraries, we'd be able to potentially support them as well. Our current implementation is pretty tightly coupled to React, so this would take a bit of work, but I don't see why it isn't possible. In the long term, we'd like to be able to support all platforms/frameworks.

Feel free to jump into the [wagmi discussions](https://github.com/tmm/wagmi/discussions) if you'd like to contribute a proposal for your library of choice!

**Reply by itsmnthn** [reply]
I guess this makes sense for now. since the kit depends on wagmi.

**Reply by itsmnthn** [reply]
I was exploring things on the side of just ConnectWallet lib that supports multiple wallet, networks and accounts. All of this can be configured and has reactive independent store(e.g. [RxJS](https://rxjs.dev/) ) that has few actions and reactive properties which can be `core` and using this core we can have multiple UI for multiple frameworks separately. This way projects can also design their own UI.
